### PR TITLE
Responsive font-sizes implementation

### PR DESCRIPTION
**Pull request by MartijnCuppens**
Implementation of responsive font-size system [RFS](https://github.com/MartijnCuppens/rfs/).

Link to feature-request:
https://github.com/twbs/bootstrap/issues/23053

CC: @XhmikosR, @kimamil, @mdo

### CODE REVIEW

**Comment by mabar** [comment]
[Poly fluid sizing](https://www.smashingmagazine.com/2017/05/fluid-responsive-typography-css-poly-fluid-sizing/#the-holy-grail-of-css-sizing) would be better. More fluent.

**Comment by MartijnCuppens** [comment]
@mabar, this implementation doesn't require to define mappings for every font-size so it can easily be used by users who do not have much frontend experience.

**Review by kimamil — CHANGES_REQUESTED** [review]
Can you add an example in the documentation on how to use this with rem? As I understood, rfs must be 'false' and unit set to 'rem'. Then I could use media-breakpoints on the html tag to set the font-size in 'rem'? But basically you suggest using fixed px sizes to set the base font-size in all breakpoints, correct?

Else, nice catch!

File: docs/4.0/content/responsive-font-sizes.md [hunk 1]
```diff
@@ -7,48 +7,111 @@ toc: true
 ---
 
 ## Responsive font sizes
-Bootstrap uses <a href="https://github.com/MartijnCuppens/rfs" target="_blank" rel="noopener">RFS</a> to control its font-size. RFS is an SCSS-mixin which automatically calculates the correct font size based on the browsers screen width. You just have got to define your font-size for big screens and the font size will automatically decrease for smaller screens.
+Bootstrap uses <a href="https://github.com/MartijnCuppens/rfs" target="_blank" rel="noopener">RFS</a> to control its font size. RFS is an SCSS-mixin which automatically calculates the correct font size based on the browsers screen width. You just have got to define your font size for big screens and the font size will automatically decrease for smaller screens.
```
Comment (XhmikosR) [inline docs/4.0/content/responsive-font-sizes.md]: Please change this to a normal markdown link. Same for other ones too if any.

**Comment by MartijnCuppens** [comment]
@kimamil, the previous documentation might indeed be a little bit confusing since it didn't explain how the configuration works but explained how you disable RFS and use rem-units for font-resizing just like you can do now.

I've updated the documentation and tried to explain how every setting of the configuration changes the way the responsive font size is calculated. Is this a bit clear or does anyone has any suggestions on how to clarify this even more?

File: docs/4.0/content/responsive-font-sizes.md (hunk 1 above)
Comment (MartijnCuppens) [inline docs/4.0/content/responsive-font-sizes.md]: Done: https://github.com/twbs/bootstrap/pull/23157/commits/aa676531a18d3a099de676e9cfcbdad6fb205ed1

**Pull request closed without merging**